### DISCUSSION TITLE
Helper for splitting room

### DISCUSSION THREAD

**Discussion by xav4nte** [rand256/valetudo#365] [Ideas]
Would it be possible to add a "helper" in the UI for editing rooms. I have a really hard time succeeding in splitting my rooms. I've seen to have some more luck by taking the x/y-values posted by Valetudo and making some slight modifications and run the mirobo split_segment command manually.
What really would help in this case is if it was possible to see the X/Y-coordinate values for a selected point on the map. Even just a console.log (like on the segments) would help a lot.

**Comment by rand256** [comment]
When you're on the map tab, you can open browser Console and call `map.getLocations().gotoPoints` to see coordinates of Goto Point (red dot) placed on the map.

**Reply by dexsta** [reply]
@rand256
Hello my friend, awesome, awesome job you have put into this project; it is appreciated, thank you. Please excuse my dumbness; I'm trying to use this and send my Bots (Gen1 & Gen2) to a bin location. When I publish this command nothing happens. Am I doing something wrong ? 

Once again thank you.

![1](https://user-images.githubusercontent.com/41833996/145510477-cd3e023a-9841-42b1-9047-39cd59687b91.png)
![2](https://user-images.githubusercontent.com/41833996/145510486-ffe4057b-dfb1-4672-94bd-e71bb3d26ff4.png)

**Reply by pidator** [reply]
You're using the wrong topic. See [the Wiki with available topics](https://github.com/rand256/valetudo/wiki/MQTT-API#Topic_valetudorockrobocustom_command), you'll need the `custom_command` topic with the `go_to` command for your action.

**Reply by dexsta** [reply]
> You're using the wrong topic. See [the Wiki with available topics](https://github.com/rand256/valetudo/wiki/MQTT-API#Topic_valetudorockrobocustom_command), you'll need the `custom_command` topic with the `go_to` command for your action.

Nice; anyone trying the same, this works for me.   Thanks for pointing me in the right direction.

![4](https://user-images.githubusercontent.com/41833996/145531646-5ead535f-04fd-4d97-8ee0-36dac26e9360.png)
